tests/release/conftest.py

```diff
@@ -0,0 +1,158 @@
+import functools
+import os
+import time
+from collections.abc import Callable
+from pathlib import Path
+from typing import Any
+
+import pandas as pd
+import pytest
+from memory_profiler import memory_usage  # type: ignore
+
+from pseudopeople.interface import (
+    generate_american_community_survey,
+    generate_current_population_survey,
+    generate_decennial_census,
+    generate_social_security,
+    generate_taxes_1040,
+    generate_taxes_w2_and_1099,
+    generate_women_infants_and_children,
+)
+
+DATASET_GENERATION_FUNCS: dict[str, Callable[..., Any]] = {
+    "census": generate_decennial_census,
+    "acs": generate_american_community_survey,
+    "cps": generate_current_population_survey,
+    "ssa": generate_social_security,
+    "tax_w2_1099": generate_taxes_w2_and_1099,
+    "wic": generate_women_infants_and_children,
+    "tax_1040": generate_taxes_1040,
+}
+
+DEFAULT_YEAR = 2020
+DEFAULT_STATE = None
+DEFAULT_POP = "sample"
+FULL_USA_FILEPATH = "/mnt/team/simulation_science/pub/models/vivarium_census_prl_synth_pop/results/release_02_yellow/full_data/united_states_of_america/2023_08_21_16_35_27/final_results/2023_08_31_15_58_01/pseudopeople_simulated_population_usa_2_0_0"
+RI_FILEPATH = "/mnt/team/simulation_science/pub/models/vivarium_census_prl_synth_pop/results/release_02_yellow/full_data/united_states_of_america/2023_08_21_16_35_27/final_results/2023_08_31_15_58_01/states/pseudopeople_simulated_population_rhode_island_2_0_0"
+SOURCE_MAPPER = {"usa": FULL_USA_FILEPATH, "ri": RI_FILEPATH, "sample": None}
+DEFAULT_ENGINE = "pandas"
+
+
+def pytest_addoption(parser: pytest.Parser) -> None:
+    parser.addoption(
+        "--dataset",
+        action="store",
+        help="The dataset to generate. Options are census, acs, cps, ssa, tax_w2_1099, wic, and tax_1040.",
+    )
+    parser.addoption(
+        "--population",
+        action="store",
+        default=DEFAULT_POP,
+        help="The simulated population to generate. Options are USA, RI, and sample. sample will generate very small sample data.",
+    )
+    parser.addoption(
+        "--engine",
+        action="store",
+        default=DEFAULT_ENGINE,
+        help="The engine used to generate data. Options are pandas and dask.",
+    )
+    parser.addoption(
+        "--state",
+        action="store",
+        default=DEFAULT_STATE,
+        help="The state to subset our data to (if using full USA population). None means no subsetting will be done.",
```

Comment:
Add state label examples (i.e. 'RI' or 'Rhode_Island' or 'Rhode Island' or... I honestly don't recall. I think it's 2-character state codes)?